### ISSUE TITLE
Replace custom initializers

### ISSUE THREAD

**Issue by mkllnk** [openfoodfoundation/ofn-install#444]
The Australian instance uses a custom initializer to activate the Connect And Learn feature toggle:

```rb
require 'open_food_network/feature_toggle'

module OpenFoodNetwork
  FeatureToggle.class_eval do
    private

    def self.features
      {order_cycles: true,
       connect_learn_homepage: true}
    end
  end
end
```

This initializer would be overridden by an ofn-install deployment. Now we have several options:

1. Enable ofn-install to store custom initializers.
2. Add a new environment variable `OFN_FEATURE_CONNECT_LEARN` to ofn-install and change the `FeatureToggle` class to use the variable.
3. Enable admins to set custom HTML on the homepage and remove the FeatureToggle code.
4. Replace the homepage with our custom Wordpress page.

I think that we don't want to wait for Wordpress. Enabling custom HTML would be obsolete when we switch to a Wordpress homepage. Using an environment variable needs changes in two repositories. It also changes the FeatureToggle class which will soon be obsolete. Option 1 - custom initializers seems to be the only option that could be beneficial in the future. Another use case is an initializer like this:

```rb
class LogBeforeTimeout
  def initialize(app)
    @app = app
  end

  def call(env)
    thr = Thread.new do
      sleep(120 - 1) # set this to Unicorn timeout - 1
      unless Thread.current[:done]
        path = env["PATH_INFO"]
        qs = env["QUERY_STRING"] and path = "#{path}?#{qs}"
        Rails.logger.warn "#{path} I’m about to timeout bringing down my unicorn worker too :("
      end
    end
    @app.call(env)
  ensure
    thr[:done] = true
    thr.run
  end
end
```

I recommend option 1.

**Comment by sauloperez** [comment]
I'd try not to move on the opposite direction of https://github.com/openfoodfoundation/ofn-install/issues/188. Initializers are app code and such need to be with the rest of it.

> Using an environment variable needs changes in two repositories. It also changes the FeatureToggle class which will soon be obsolete.

I actually find this an upside. Whenever we deprecate FeatureToggle we'll easily know this needs to updated as well, whereas being in ofn-install is very likely that we forget about it.

**Comment by mkllnk** [comment]
Will we still have Feature Toggles in the future? The Connect-And-Learn toggle is the only one at the moment and we would like to get rid of it as well.

**Comment by Matt-Yorkley** [comment]
Closed by #462
